addons/website_slides/views/slide_slide_views.xml

```diff
@@ -40,19 +48,51 @@
             <field name="arch" type="xml">
                 <form string="Lesson">
                     <sheet>
+                        <field name="channel_type" invisible="1" readonly="1"/>
+                        <field name="channel_allow_comment" invisible="1" readonly="1"/>
                         <div class="oe_button_box" name="button_box">
-                            <field name="is_published" widget="website_redirect_button"
+                            <button disabled="1" class="mx-auto" icon="fa-eye">
+                                <span class="o_stat_value">
+                                    <field name="slide_views"/> Member Views
+                                </span>
+                            </button>
+                            <button disabled="1" class="mx-auto" icon="fa-eye">
+                                <span class="o_stat_value">
+                                    <field name="public_views" readonly="1"/> Public Views
+                                </span>
+                            </button>
+                            <button disabled="1" class="mx-auto" icon="fa-eye">
+                                <span class="o_stat_value">
+                                    <field name="total_views"/> Total Views
+                                </span>
+                            </button>
+                            <button disabled="1" attrs="{'invisible': [('channel_type', '=', 'training')]}" icon="fa-thumbs-up">
+                                <span class="o_stat_value" >
+                                    <field class="ml-1" name="likes" string="Likes"/> Likes
+                                </span>
+                             </button>
+                             <button disabled="1" attrs="{'invisible': [('channel_type', '=', 'training')]}" icon="fa-thumbs-down">
+                                 <span class="o_stat_value">
+                                    <field class="ml-1" name="dislikes"/> Dislikes
+                                 </span>
+                             </button>
+                             <button disabled="1" attrs="{'invisible': [('channel_allow_comment', '!=', True)]}" icon="fa-comments">
+                                <span class="o_stat_value">
+                                    <field class="ml-1" name="comments_count"/> Comments
+                                </span>
+                            </button>
+                            <field name="is_published" style="margin-left:10px" widget="website_redirect_button"
                                    attrs="{'invisible': ['|',('is_category', '=', True), ('channel_id', '=', False)]}"/>
```

Comment:
Still not solved (style attribute)

Reply:
My bad I didn't see this one, I am rectifying this.